python/paddle/fluid/layers/nn.py

```diff
@@ -5036,6 +5037,45 @@ def gather(input, index):
     return out
 
 
+def scatter(input, index, updates):
```

Comment:
```
def scatter(input, index, updates, name=None):
```

Please also add unit tests in https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/unittests/test_layers.py

Reply:
Done